test/harry/main/org/apache/cassandra/harry/model/ASTSingleTableModel.java

```diff
@@ -517,6 +520,8 @@ private void update(Mutation.Delete delete)
             DeleteKind kind = DeleteKind.PARTITION;
             if (!delete.columns.isEmpty())
                 kind = DeleteKind.COLUMN;
+            else if (containsRangeCondition(split.right))
```

Comment:
nit: if the range isn't on the clustering key, then its not a range tombstone... maybe rename to be clear this only matters for clustering?